app/helpers/control_objective_items_helper.rb

```diff
@@ -50,4 +50,18 @@ def control_objective_weaknesses_link(control_objective_item)
   def auditor_comment_options
     CONCLUSION_OPTIONS.map { |option| [option, option] }
   end
+
+  def link_to_recover_original_control_objective_name(control_objective_item)
+    link_to(
+      content_tag(:span, nil, class: 'glyphicon glyphicon-warning-sign text-danger'),
```

Comment:
Esto no se hace, en la documentación de bootstrap dice que no hay que usar otras clases en el ```<span>``` del icono: https://getbootstrap.com/docs/3.3/components/#callout-glyphicons-dont-mix

Reply:
Si si... dice `cannot be directly combined ` pero anda igual, así que ... `s/cannot/shouldn't/` jaja